spec/rubocop/cop/performance/collection_literal_in_loop_spec.rb

```diff
@@ -269,4 +269,74 @@
       RUBY
     end
   end
+
+  context 'when Ruby >= 3.4', :ruby34 do
+    # TODO: Remove when the minimum supported RuboCop is 1.62.0
+    let(:ruby_version) { 3.4 }
```

Comment:
The current latest version of RuboCop Performance already requires RuboCop 1.72+. Can you remove it?

Reply:
Nice, I removed it